lib/apia/rack.rb

```diff
@@ -79,7 +79,8 @@ def handle_request(env, api_path)
 
       validate_api if development?
 
-      route = find_route(request_method, api_path)
+      access_control_request_method = env['HTTP_ACCESS_CONTROL_REQUEST_METHOD']&.upcase
+      route = find_route((access_control_request_method || request_method), api_path)
```

Comment:
In the options request it tells us the request method of the actual API request in `HTTP_ACCESS_CONTROL_REQUEST_METHOD`, so we use that to find the corresponding route. Otherwise we just get a 404.

Reply:
I think we should only use this header when requests themselves are OPTIONS.